R/integer64.R

```diff
@@ -556,11 +562,11 @@ identical.integer64 <- function(x, y,
 
 #' @rdname as.integer64.character
 #' @export
-as.integer64 <- function(x, ...) UseMethod("as.integer64")
+as.integer64 = function(x, keep.names=FALSE, ...) UseMethod("as.integer64")
```

Comment:
which coercion methods are you having in mind for this argument? IMO it's a bit inconsistent and users should always expect `as.*` coercions to strip names -- `as.{raw,logical,integer,double,numeric,complex,character}` all do, and those are the methods we should be imitating.